src/index.js

```diff
@@ -34,6 +34,26 @@ export function toggleCreator(context) {
   }
 }
 
+export function handleSumbit(func){
+  return (e) => {
+    e.preventDefault();
+
+    const form = e.target;
+    let values = {};
+
+    let children = [];
+
+    children.push(form.querySelectorAll('input, select, textarea'));
```

Comment:
I tested it again this morning and I wansn't getting buttons... 
worse case scenario, the values object will have a hanging `"":""`